src/parser.rs

```diff
@@ -37,6 +37,44 @@ use crate::ParsedDaysOfWeek;
 use crate::PossibleLiterals;
 use crate::PossibleValue;
 
+/// Options to manipulate the parsing manner.
+#[derive(Debug, Copy, Clone)]
+pub struct ParseOptions {
```

Comment:
FYI, adding a new member to this struct in the future will be a semver incompatible change.

Reply:
I remember adding a private field (`_priv: ()`) can prevent this?

Reply:
clippy hints me to add `#[non_exhaustive]` for this case.

Ref - https://rust-lang.github.io/rust-clippy/master/index.html#/manual_non_exhaustive